config/locales/server.en.yml

```diff
@@ -1702,6 +1702,7 @@ en:
     enable_emoji_shortcuts: "Common smiley text such as :) :p :( will be converted to emojis"
     emoji_set: "How would you like your emoji?"
     enforce_square_emoji: "Force a square aspect ratio to all emojis."
+    emoji_trigger: "Minimum number of characters required to trigger autocomplete emoji popup"
```

Comment:
I think this name can be improved a bit `emoji_autocomplete_min_chars`  except for that, I know we don't have many tests to go by here, but I think it would be interesting to add one. Not going to block the PR on it, but if you can figure this out it would be very cool.

Reply:
I have been trying to figure out how to do the test.

```javascript
componentTest("minimum characters for emoji autocomplete == 0", {
  template: "{{d-editor value=value}}",

  beforeEach() {
    this.siteSettings.emoji_autocomplete_min_chars = 0;
  },

  async test(assert) {
    await this.set("value", ":");
    assert.equal(this.$(".autocomplete.ac-emoji").length, 1);
  }
});

componentTest("minimum characters for emoji autocomplete == 1", {
  template: "{{d-editor value=value}}",

  beforeEach() {
    this.siteSettings.emoji_autocomplete_min_chars = 1;
  },

  async test(assert) {
    await this.set("value", ":");
    assert.equal(this.$(".autocomplete.ac-emoji").length, 0);

    await this.set("value", ":s");
    assert.equal(this.$(".autocomplete.ac-emoji").length, 1);
  }
});
```

This is what I got, but for some reason it does not work. Do you have idea?